cmd/buildah/commit.go

```diff
@@ -26,6 +27,10 @@ var (
 			Name:  "format, f",
 			Usage: "`format` of the image manifest and metadata",
 		},
+		cli.StringFlag{
+			Name:  "reference-time",
+			Usage: "set the timestamp on the image to match the named `file`",
```

Comment:
If this is for self tests, can we hide the option and not document or command completion it.